source/gent/tier1_hortense.rst

```diff
@@ -216,28 +221,102 @@ This storage space can be addressed with the environment variable $TMPDIR
   cd $TMPDIR
 
 
-Project storage
-***************
+.. _hortense_home_on_scratch:
+
+Home-on-scratch setup
+*********************
+
+On Tier-1 Hortense, the home directory (``$HOME``) corresponds to your personal scratch directory (``$VSC_SCRATCH``),
+rather than your usual VSC home directory (``$VSC_HOME``).
+
+This is done to ensure that Tier-1 Hortense can remain operational, even if there is maintenance being
+performed on the Tier-2 shared storage filesystem of a VSC site (UGent, KUL, VUB, UAntwerpen),
+or in case of problems with the network connection to the other VSC sites.
+
+Although your VSC home directory is usually accessible via ``$VSC_HOME``,
+we strongly recommend to *not* simply create symbolic links to files like your ``.bashrc`` startup script,
+since that would defeat the purpose of this "home-on-scratch" setup.
+
+This recommendation also applies to ``$VSC_DATA``: you should avoid using it in your job scripts as much as
+possible, and ensure that your workflow only relies on the Hortense scratch filesystem. If you require any
+data as input for your jobs, it should be copied to the Hortense scratch filesystem first (for example
+via :ref:`globus`).
```

Comment:
i would not mention only globus; it's overkill for small things and eg industry people don't have access to it